ee/WFLY-19776_Jakarta_Data_in_standard_WildFly.adoc

```diff
@@ -0,0 +1,137 @@
+
+=== User Stories
+
+I'm an application developer and I'd like to be less concerned with the mechanics of manipulating persistent data. I'd like to define my entities and then write an interface that describes the CRUD and data access operations I need. I code against that interface and let WildFly take care of generating the interface implementation that deals with the DB.  I'm fine with having less flexibility with the portions of my app that work this way, including limitations like:
+* A simpler query language.
+* No lazy loading  -- or only explicit lazy loading by using Hibernate ORM native APIs.
+* More limited support for dynamic queries than is available with alternatives like the Criteria API.
+* And so on.
+I recognize that for portions of my application where these limitations are unacceptable I can still use standard Jakarta Persistence or native Hibernate ORM APIS.
+
+I'm an application developer and am familiar with the Repository pattern from use of Spring Data. I'd like to use that pattern in WildFly in non-Spring applications.
+
+I'm an application developer who's used Deltaspike Data, which uses a repository pattern. But it doesn't support EE > 8, so I'm looking for a similar technology to run on the latest WildFly releases.
+
+== Issue Metadata
+
+=== Related Issues
+
+https://issues.redhat.com/browse/WFLY-19776
+
+=== Affected Projects or Components
+
+https://github.com/wildfly/wildfly
+
+=== Other Interested Projects
+
+https://github.com/hal/console
+
+=== Relevant Installation Types
+
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* Managed domain
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+
+== Requirements
+
+* The JBoss Modules modules required for the `jakarta-data` subsystem and the Jakarta Data API will be made available through `wildfly-ee` feature pack.
+** These modules will be shared with the `wildfly-preview` feature pack. (This is the norm for `wildfly-ee` modules).
+* The `jakarta-data` Galleon layer will be made available through the `wildfly-ee` feature pack.
+** This layer will be shared with the `wildfly-preview` feature pack. (This is the norm for `wildfly-ee` layers).
+* The Jakarta Data API artifact *must not* be part of the standard WildFly user boms. We limit the user boms to `community` stability artifacts.
+* The Jakarta Data API artifact must contintue to be part of the WildFly Preview user boms.
+* The wildfly-tck-runners Jakarta Data runner will be modified to test the `wildfly` feature pack by default, while allowing testing of `wildfly-preview`.
```

Comment:
Note: I'll do this after the 35 Beta release as the runner shouldn't depend on a SNAPSHOT.